zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -720,15 +770,15 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, environment, region,
             'shards': {
                 'postgres': '{}:{}/postgres'.format(service_dns_name, POSTGRESQL_DEFAULT_PORT)
             },
-            'expected_replica_count': ss.get('replicas', 0),
```

Comment:
This should also not change, we overwrite min replica count and it should stick.